lib/Textmaster/Model/Document.php

```diff
@@ -206,6 +206,19 @@ public function getSourceContent()
     /**
      * {@inheritdoc}
      */
+    public function getTranslatedContent()
+    {
+        $authorWork = $this->getProperty('author_work');
+        if (self::TYPE_STANDARD === $this->getType() && !empty($authorWork)) {
+            return $authorWork['free_text'];
+        }
+
+        return $this->formatTranslatedContent();
```

Comment:
As it is deprecated, it think that the content of this method could be replaced just by
```
return $this->getSourceContent();
```

The main point of a BC is when a method is totally removed from an interface. Changing what the method is not à BC break.